ocs_ci/ocs/ui/base_ui.py

```diff
@@ -334,9 +333,11 @@ def get_page_hash():
             time.sleep(sleep_time)
             page_hash_new = get_page_hash()
             if retry_counter == retries:
-                raise PageNotLoaded(
+                logger.error(
```

Comment:
Is it intention here to not raise exception if retry_counter == retries and just do logger.error instead?
Will the code continue as I think the exception prevent this before and fail?

Reply:
Correct.. if we can't click on a specific button. Selenium will raise an exception.